samples/dashboards/kpi-component/formula/demo.js

```diff
@@ -5,11 +5,11 @@ Dashboards.board('container', {
             type: 'CSV',
             options: {
                 csv: `Value
-                100
-                200
-                300
-                400
-                500`
+                10
+                20
+                30
+                40
+                50`
```

Comment:
Test with non-sorted values and NaNs.

Reply:
Sure, the new changes cover the number formatting and NaN values filtering. Also, the values are externally sorted if needed (median) so the proper format lands inside the formula functions.